setup.py

```diff
@@ -165,4 +165,16 @@
             'sample=sample:main',
         ],
     },
+
+    # List additional URLs that are relevant to your project as a dict.
+    #
+    # This field corresponds to the "Project-URL" metadata fields:
+    # https://packaging.python.org/specifications/core-metadata/#project-url-multiple-use
+    #
+    # Examples listed include a pattern for specifying where the package
+    # maintainers would like thanks given or funding support.
+    project_urls={  # Optional
+        'funding': 'https://donate.pypi.org',
+        'thanks': 'http://saythanks.io/to/example'
+    },
```

Comment:
One thing to note here is that the key is what's used to render the link text on PyPI. I think it would be ideal to make it clear to a user that they can make this whatever they want, by making the examples looks more human-friendly. It'd also be great to include some of the more common `project_urls` that this is used for, maybe something like:
```
    project_urls={  # Optional
        'Bug Reports': 'https://github.com/pypa/sampleproject/issues',
        'Source': 'https://github.com/pypa/sampleproject/',
        'Funding': 'https://donate.pypi.org',
        'Say Thanks!': 'http://saythanks.io/to/example'
    },
```
If I understand correctly, you're hoping for some consistency with the project keys here to find projects that support saying thanks, but since it's really not enforceable, a better approach might be to just look at all the `project_urls` values and see if any match what you're looking for.

Reply:
@di I can definitely add more links! I agree that these aren't enforceable, but I'm hoping good docs and usage patterns will encourage some consistency.